### DISCUSSION TITLE
How can I run qalc (preffered) or Qalculate on an android phone?

### DISCUSSION THREAD

**Discussion by conradRz** [Qalculate/libqalculate#398] [Q&A]
As in the topic. I'm looking for a way which will be fast (no emulation then?)

I use qalc a lot on PC and before that I used Qalculate. This is because of the finding for _x_ functionality as well as PV/NPV or even details such as the use of ANS. Standard Android calculator is not enough for me.

**Comment by k4leg** [comment]
You can run qalc in Termux (just install qalc, `pkg install qalc`).

**Comment by vlad2305m** [comment]
https://flaviutamas.com/qalculate-wasm/ has limited features but supports plots (#225)

**Comment by vlad2305m** [comment]
Qalculator in play store is wounderful

**Comment by abrasive** [comment]
I'm a bit late here but I've been using Termux for this for ages now! `pkg install qalc` will get you there.

Then you can use Termux:Widget to create a shortcut straight from your home screen to jump into qalc. It's fantastic to have this close at hand!

**Comment by ByteHamster** [comment]
There is an Android app that compiles qualculate on Android: https://github.com/jherkenhoff/qalculate-android
